### DISCUSSION TITLE
求助：About format of 'FE_ SERVERS' when Docker Deployment

### DISCUSSION THREAD

**Discussion by cl0924** [apache/doris#20571] [A - General / Q&A]
I am a novice. When using the official Docker Run deployment method, FE_HOST of FE_SERVERS Can HOST only be in IP format? It seems that I will report an error when using a domain name(Due to possible IP changes, I would like to use a domain name like fe-master: 9010 for deployment).Can deployment be done without using a real IP address?

`[ERROR] [Entrypoint]: FE_ SERVERS rule error! Example: $FE_ NAME: $FE_ HOST_ IP: $FE_ EDIT_ LOG_ PORT [, $FE-NAME: $FE_HOST.IP: $FE-EDIT_LOG_PORT]`

My running statement is as follows:
```
docker run -itd \
--name=fe-master \
--env FE_SERVERS="fe1:fe-master:9010" \
--env FE_ID=1 \
-p 8030:8030 \
-p 9030:9030 \
-v /data/fe/doris-meta:/opt/apache-doris/fe/doris-meta \
-v /data/fe/conf:/opt/apache-doris/fe/conf \
-v /data/fe/log:/opt/apache-doris/fe/log \
--ip= fe-master  \
apache/doris:1.2.2-fe-x86_64
```

**Comment by yagagagaga** [comment]
Doris version 1.2 currently only supports IP format.

**Reply by cl0924** [reply]
Thank you for your answer. I saw that the official version of 2.0 has been released, but after trying to modify it, it seems that it still doesn't work. Have you confirmed which version can support this format？
![image](https://github.com/apache/doris/assets/84334635/318f27b6-4ade-4c49-9f0f-167b1c0c72bc)

**Reply by yagagagaga** [reply]
As far as I know, the current Docker images could not support domain names, but you can create your own images. Here is the [Dockerfile](https://github.com/apache/doris/tree/master/docker/runtime/be) you can refer to.
